extensions/mssql/src/hdfs/webhdfs.ts

```diff
@@ -340,11 +340,11 @@ export class WebHDFS {
 	}
 
 	/**
-	 * Read directory contents
+	 * List the status of a path
 	 *
 	 * @returns void
 	 */
-	public readdir(path: string, callback: (error: HdfsError, files: any[]) => void): void {
```

Comment:
So the reason for these names was to show a "file system API compatible" wrapper on the WebHDFS module. That's the design of the upstream module that we forked and I kind of like it. Also in the long-term, I would love to be able to contribute back since I've seen folks asking for Kerberos support in that module. Given this:
- Are we sure we want to rename this?
- Are we sure we want to remove stat (which is another filesystem operation API)?

Reply:
This is just the direct wrapper over the WebHDFS calls. We still have HdfsFileSource in fileSources that is the "file system API compatible wrapper" and still has the common names such as stat and readdir. I wanted to update the names so that we can easily see what WebHDFS methods have been implemented and then HdfsFileSource would call the ones it needs to fulfill the contract.